site/get-started.markdown

```diff
@@ -0,0 +1,118 @@
+
+Welcome, new Haskeller! Read on to quickly set up your Haskell development environment, execute your first lines of code, and get directions for further learning.
+
+## Content
+  - [Set up a Haskell development environment](#set-up-a-haskell-development-environment)
+    - [GHCup: universal installer](#ghcup-universal-installer)
+    - [Editor](#editor)
+  - [Running your first lines of code](#running-your-first-lines-of-code)
+  - [Writing your first Haskell program](#writing-your-first-haskell-program)
+  - [Participate in the community](#participate-in-the-community)
+  - [Next steps](#next-steps)
+
+## Set up a Haskell development environment
+
+A complete Haskell development environment consists of the Haskell toolchain (compiler, language server, build tool) and an editor with good Haskell support. The quickest way to get this set up is to:
+
+1. Use **GHCup** to install and manage the Haskell toolchain.
+2. Use [**VSCode**](https://code.visualstudio.com/) as the editor, with the Haskell extension installed.
+
+### GHCup: universal installer
+
+[GHCup](https://www.haskell.org/ghcup/#) is a universal installer for Haskell that will install everything you need to program in Haskell, and will help you manage those installations (update, switch versions, ...). Follow the instructions on the [GHCup webpage](https://www.haskell.org/ghcup/#) to install GHCup. Then use it to install the Haskell toolchain, which consists of:
+
+1. **GHC** (The Haskell compiler) We will use GHC below to run our examples, but in practice you will mostly use a tool like Cabal or Stack to build your code, instead of GHC directly.
+2. **HLS** (The Haskell Language Server) You won't use HLS directly, instead your code editor will use it in the background to provide you with a great experience while editing Haskell code.
+3. **Cabal** (A Haskell build tool) You will use Cabal to structure your Haskell projects, build them, run them, define dependencies, ... .
+4. **Stack** (A Haskell build tool) An alternative to Cabal.
+
+
+<div class="bs-callout bs-callout-info">
+  <p>
+    <h4>Cabal and Stack: which one should I install?</h4>
+    We recommend installing both. Most Haskell projects can be built using Cabal, but some might require Stack. Installing both guarantees that you can use either, and while following a tutorial or book you can use whatever they recommend.
+  </p>
+</div>
+
+### Editor
+[**Visual Studio Code**](https://code.visualstudio.com/) ([**VSCode**](https://code.visualstudio.com/)) is a popular choice with well-supported Haskell integration. Install the [Haskell extension](https://marketplace.visualstudio.com/items?itemName=haskell.haskell) and you are all set. It should work out of the box and use your installation of HLS. To learn about support for other editors, check out [HLS docs for editor configuration](https://haskell-language-server.readthedocs.io/en/latest/configuration.html#configuring-your-editor).
+
+## Running your first lines of code
```

Comment:
I wonder if we could bring the interactive tutorial over from the main page to here to make some of this a bit more interactive?

Reply:
I think that's probably worth delaying to a separate PR.

Reply:
That is interesting idea! I was considering a bit what to do with it, since it feels like right now we have two CTAs on landing page -> one is "Get Started" button, second is "Try it" prompt. Ideally there would be just one. But, if we would try to include it in the get-started page, I think it would side track us -> on get-started we are getting them to install tooling, and then we want them to use what they installed and try haskell through that, so when they are done with the get-started page, they have everything installed and have an idea how to use it -> they now feel ready to play with Haskell on their machine and had their first victory.

So at the moment I don't have a good idea how to plug in Try It into get-started. One option might be to remove it from landing page, but that does feel a bit bad since it is so cool. So I would probably just leave it for the time being, I don't think it hurts.